vscode/src/chat/MessageProvider.ts

```diff
@@ -570,18 +570,18 @@ export abstract class MessageProvider extends MessageHandler implements vscode.D
                 return vscode.commands.executeCommand('cody.action.commands.menu')
             case text === '/commands-settings':
                 return vscode.commands.executeCommand('cody.settings.commands')
-            case /^\/o(pen)?/i.test(text) && this.editor.controllers.command !== undefined:
+            case /^\/o(pen)?\s.*$/.test(text) && this.editor.controllers.command !== undefined:
```

Comment:
```suggestion
            case /^\/o(pen)?\s/.test(text) && this.editor.controllers.command !== undefined:
```

I think this should be equivalent, wdyt?